.github/workflows/check-package-managers.yml

```diff
@@ -0,0 +1,27 @@
+name: Check package managers
+on: [pull_request]
```

Comment:
I wonder if instead of having this run on every PR (& PR commit), we just have it run on every merge/push to master? Just so it doesn't slow down too much the test runs on each commit.

Reply:
Yeah, I wanted to test out how long this would take, and right now the package manager checks are actually faster than the test workflow. Since they're running in parallel, running the package manager checks on PRs won't impact the total time.

That being said, I'd like to add a check for SPM, so we'll have to see how fast that runs. :)